pkgs/by-name/mo/modrinth-app/package.nix

```diff
@@ -33,38 +34,55 @@ symlinkJoin rec {
     glib-networking
   ];
 
-  nativeBuildInputs = [
-    wrapGAppsHook3
-  ];
+  nativeBuildInputs = [ wrapGAppsHook3 ];
+
+  runtimeDependencies = lib.optionalString stdenv.isLinux (
+    lib.makeLibraryPath [
+      addDriverRunpath.driverLink
+
+      # glfw
+      libGL
+      xorg.libX11
+      xorg.libXcursor
+      xorg.libXext
+      xorg.libXrandr
+      xorg.libXxf86vm
+
+      # lwjgl
+      stdenv.cc.cc.lib
 
-  runtimeDependencies = lib.optionalString stdenv.isLinux (lib.makeLibraryPath [
-    addDriverRunpath.driverLink
-    flite # narrator support
+      flite # narrator support
 
-    udev # oshi
+      # openal
+      libpulseaudio
 
-    # lwjgl
-    libGL
-    libpulseaudio
-    stdenv.cc.cc.lib
-    xorg.libX11
-    xorg.libXcursor
-    xorg.libXext
-    xorg.libXxf86vm
-    xorg.libXrandr
-  ]);
+      udev # oshi
+    ]
+  );
 
   postBuild = ''
     gappsWrapperArgs+=(
       --prefix PATH : ${lib.makeSearchPath "bin/java" jdks}
       ${lib.optionalString stdenv.isLinux ''
-        --prefix PATH : ${lib.makeBinPath [xorg.xrandr]}
+        --prefix PATH : ${lib.makeBinPath [ xorg.xrandr ]}
         --set LD_LIBRARY_PATH $runtimeDependencies
       ''}
     )
 
     wrapGAppsHook
   '';
 
-  inherit (modrinth-app-unwrapped) meta;
+  meta = {
```

Comment:
IMO I would just keep inheriting meta and filter out what we don't want to not forget something

Reply:
I tried to use `builtins.removeAttrs` for this but it seems it still breaks `meta.position`. Not really sure of another way to still have that without explicit `inherit`s :/